doc/watch-and-regenerate.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash -eu
-cd ..
-sudo fatrace | awk '/.* CW .*\/b2-sdk-python\/.*\.(rst|py)$/ {print;system("cd doc; ./regenerate.sh")}'
```

Comment:
`nox -s doc` will do it for you :) (an interactive mode which is default one). It also watches for any changes in `b2-sdk-python`  by using `sphinx-autobuild` instead of `sphinx-build`

Reply:
great, but please provide quick hints to other developers with stuff like that in the docs - I wouldn't know I could use this feature in this project (and probably a few other useful things too)

Reply:
Added to new `CONTRIBUTING.md` file